README.md

```diff
@@ -1,19 +1,76 @@
 # Kashmir [![NPM version][npm-image]][npm-url]
-> Generator for the Kashmir Drupal theme
+> A Drupal theme generator
 
 ## Installation
 
-First, install [Yeoman](http://yeoman.io) and generator-kashmir using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+Install [Yeoman](http://yeoman.io) and generator-kashmir using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
 
 ```bash
 npm install -g yo generator-kashmir
 ```
 
-Then generate your new theme:
+Then generate your new theme using the generator. Use the below command from the directory where you want the theme to be generated.
 
 ```bash
 yo kashmir
 ```
 
 [npm-image]: https://badge.fury.io/js/generator-kashmir.svg
 [npm-url]: https://npmjs.org/package/generator-kashmir
+
+## Design systems
+
+The theme provides two options for design systems, storybook and patternlab. These are included as part of the theme and you can choose one of the two during the theme generation process.
```

Comment:
We should fix the casing of Storybook and Patternlab. Also, we should add links to their websites.